.travis.yml

```diff
@@ -2,28 +2,21 @@ language: python
 
 python:
   -3.4.5
-# command to install dependencies
 
-- Create virtual environment
-```
-python -m venv myvenv
-```
-- Activate Virtual environment
-```
-myvenv\Scripts\activate
-```
+# command to create virtual venv
+- python -m venv venv
+
+# command to activate virtual venv
+- myvenv\Scripts\activate
 
-- Install requirements
-```
-pip install -r requirements.txt
-```
 
-## Running the application
-```
-python request.py
-```
+# command to install dependencies
+install:
+  - pip install -r requirements.txt
+        
+# command to run tests
+script: 
+  - python -m unittest
+
+  after_success: coveralls
```

Comment:
You need to install `coveralls` and `coverage` via pip. This particulary reports test coverage to coveralls